### ISSUE TITLE
make excalidraw css selectors more specific

### ISSUE THREAD

**Issue by dwelle** [excalidraw/excalidraw#3825]
Even if we're targeting selectors inside `.excalidraw`, we shouldn't use element types (`button`, `label` etc.) because it would override custom JSX that host apps may inject into the UI via props.

https://github.com/excalidraw/excalidraw/blob/0749d2c1f3902ad32f0bdaeefa36bd3b0e137dc7/src/css/styles.scss#L188-L189

/cc @ad1992

**Comment by ad1992** [comment]
Yes completely agree we shouldn't use element types, we can add a prefix to make sure its excalidraw button/dialog etc.

**Comment by firmanjml** [comment]
Hi, is there any solution to this issue? I'm currently facing issue with this being conflicted with the bootstrap and had to rename the css selectors manually to make this work.

popover and dropdown-menu selectors are the one causing the popover menu to look weird and the dropdown-menu not opening .

**Comment by e-conforti** [comment]
Any progress here? I am having the same problem like in the referenced issue, that bootstrap is changing the dropdown class

**Comment by ghost** [comment]
Any progress yet on this one? still bootstrap is messing around with hamburger menu.
